app/models/comment.rb

```diff
@@ -0,0 +1,8 @@
+class Comment < ApplicationRecord
+  belongs_to :user, class_name: 'User', foreign_key: 'author_id'
+  belongs_to :post, class_name: 'Post', foreign_key: 'post_id'
+
+  def five_most_recent_comments_for_user(user)
```

Comment:
- [ ] The comment model should include a method that updates the comments counter for a post, kindly consider implementing this feature

Reply:
Updated